### DISCUSSION TITLE
Milestones per organization

### DISCUSSION THREAD

**Discussion by askpt** [community/community#5266] [Other Feature Feedback, Questions, & Ideas]
Like any organization that has multiple repositories, we would like to have milestones that are shared across repositories. Almost like the `Projects` are currently shared. It would be great to see the PRs and issues associated with a specific milestone at the organization level (plus that interesting feature where we can see the milestone progress at the issue/PR level).

Do you think it's something that makes sense?

**Comment by erikr** [comment]
Agree, this would be extremely helpful. Having cross-repo projects is a good start but we need the granularity to group cross-repo tasks into epics (milestones) within projects!

**Comment by gtupak** [comment]
There's a very popular similar thread based on this: https://github.com/dear-github/dear-github/issues/93

**Reply by gabriel-r** [reply]
Weird, that issue has almost `1k 👍` votes while this one only `43 ⬆️`

![image](https://github.com/community/community/assets/51084/8962ee09-5dfd-4cdb-88d6-a006b15194cc)

Please GitHub overlords, make this happen.

**Reply by gtupak** [reply]
Indeed!

**Reply by jasonbvh** [reply]
> There's a very popular similar thread based on this: [dear-github/dear-github#93](https://github.com/dear-github/dear-github/issues/93)

do you know how I can update the milestones app that I deployed last yeah from github? how do I get the updated version and how do I apply it?
Users are saying it's not working and saying it needs an update?

I am new to github I am not a developer just used github to download and installed the custom app in teams . now looking for the update and how to update it?

**Comment by DrHouston** [comment]
Yes Please - I am manually entering the same milestones across repositories:)

**Reply by flavienbwk** [reply]
You can create a "Release" or "Milestone" custom project field, that's more convenient.

**Comment by f-rakete** [comment]
+1 let's keep the bumping going over here as well

**Comment by djcesar** [comment]
+1

**Comment by jasonbvh** [comment]
does anyone know how do I update the existing Milestones app that is deployed in teams, users are complaining it's not working as expected and asking for an update?

**Comment by janahaddad** [comment]
+1

**Comment by hardikdr** [comment]
+100

**Comment by flavienbwk** [comment]
Omg Microsoft/GitHub, answer your community it's been literally **10 years** we request that.
